agriculture_pricing/README.md

```diff
@@ -1,3 +1,154 @@
 # Agriculture Pricing Example
 
-Example of implementation of agriculture pricing defined [here](https://github.com/Gurobi/modeling-examples/tree/master/agricultural_pricing).
\ No newline at end of file
+Example of implementation of agriculture pricing adapted from [here](https://github.com/Gurobi/modeling-examples/tree/master/agricultural_pricing).
+
+## Data View
+
+Columns marked with __*__ are index columns.
+
+#### `capacity.csv`
+
+| `component`* | `capacity` |
+| --- | --- |
+| __Defines__ set <u>Components</u> ($C$) | __Defines__ param <U>Last Year's Demand |
+
+> Missing index values allowed? : N/A
+
+#### `composition.csv`
+
+| `dairy`* | `component`* | `composition` |
+| --- | --- | -- |
+| Elements __belong__ to <u>Dairy ($D$)</u> | Elements __belong__ to set <u>Components ($C$)</u> | __Defines__ param <u>Composition</u> |
+
+> Missing index values allowed? : No
```

Comment:
1) is this a new SQL column for the data_source table?
3) is this defined by the user or determined by some frontend logic (e.g. 'N/A' for files that are used to define source_sets, and 'No' for files that are not used to define source_sets)?

Reply:
1. Idk, I haven't thought that far. Probably?
2. It has to be N/A for any files that define source sets, and N/A for files with no index (i.e. constants.csv). And it can be either yes/no for any other file. If it's yes, then any parameters need to define a default.

Reply:
okay, i noted this down on statra notion. I think we can use the formula/logic to determine whether a missing index is allowed, because the order of operators is data sources then sets I think it would be weird to indicate N/A, no, or, yes before defining the sets.